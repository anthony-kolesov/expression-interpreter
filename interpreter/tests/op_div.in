# Typical operation.
out 6 / 3
print "\n"

# Relatively big numbers.
out 1000000 / 2
print "\n"

# Crossing the zero.
out 36 / -9
print "\n"

# Staying negative.
out -21 / 7 
print "\n"

# Floating point numbers.
out 3.9 / 1.3
print "\n"

# Floating point and integer numbers.
out 8 / 2.5
print "\n"

# Zero values.
out 0 / 1
print "\n"

# Division by one.
out 7 / 1
print "\n"

# That will be 0, because result is integer.
out 1 / 3
print "\n"

# That will be 0.5, because result is float.
out 1.0 / 2
print "\n"
