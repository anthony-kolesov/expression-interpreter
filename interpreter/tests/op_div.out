2
500000
-4
-3
3.000000
3.200000
0
7
0
0.500000
